activiti-cloud-service-common/activiti-cloud-services-dbp-rest/src/test/java/org/activiti/cloud/alfresco/filter/TrailingSlashRedirectFilterTest.java

```diff
@@ -61,18 +58,14 @@ void testGreetingWithRequestParams() throws Exception {
 
     @Test
     void testGreetingTrailingSlashWithFilter() throws Exception {
-        mvc
-            .perform(get(BASEURL + "/greeting/").accept(APPLICATION_JSON_VALUE))
-            .andExpect(status().isOk())
-            .andExpect(content().string("Hello, World!"));
+        mvc.perform(get(BASEURL + "/greeting/").accept(APPLICATION_JSON_VALUE)).andExpect(status().isNotFound());
```

Comment:
This highlights a breaking change, right? (That kind of URL is not supported anymore?)

Reply:
Same question as Anahide. I remember you added some mechanism that could enable / disable it during the upgrade to SB 3.1, to mitigate the breaking change from Spring. Do we want stop doing it now or is it no longer possible?

Reply:
This was put in place to be able to revert back to the legacy trailing path slash `/` redirect using custom property. It has always been disabled by default and there were no problems reported since we upgraded to SB v3.1.x This method will be removed in Spring Framework 6.2, so it does not make sense to keep the legacy override anyway.